### DISCUSSION TITLE
Jailbreak specific default repo shouldn’t be removable

### DISCUSSION THREAD

**Discussion by kyondev** [wstyres/Zebra#1541] [Ideas]
**Describe the bug**

The jailbreak specific default Repo can be removed. (e.g. Binger/Elucubratus repo for checkra1n)

**To Reproduce**

Try to remove the default repo

**Expected behavior**

The option to delete should be missing or not working on that specific repo like for the stable Zebra repo

**Screenshots**

**Device Information:** 

 - iOS Version: 12.4.8
 - Zebra Version: 1.2~beta4c
 - Device Type: iPhone7,2/iPhone 6
 - Jailbreak Type: checkra1n

**Crash Log or System Log**

**Additional context**

**Comment by wstyres** [comment]
This isn't a bug so we'll consider it a feature request.

The reason Zebra doesn't currently have this is because we add no "default repos". We're going to be merging sources.lists soon (#1369) so jailbreaks will be able to provide their own default repos.
